apps/web/src/pages/workflow/components/import-modal/components/style.less

```diff
@@ -0,0 +1,64 @@
+.@{prefix}-import-box {
+    .@{prefix}-import-file {
+        display: flex;
+        flex: 1 0 0;
+        flex-direction: column;
+        gap: 10px;
+        align-items: center;
+        align-self: stretch;
+        justify-content: center;
+        padding: 35px 10px 41px;
+        background: var(--gray-1);
+        border: 1px dashed var(--border-color-base);
+        border-radius: 4px;
+        cursor: pointer;
+    }
+    .@{prefix}-import-drag {
+        border-color: var(--purple-8);
+    }
+    .@{prefix}-import-error {
+        border-color: var(--mui-palette-error-main);
+    }
+    .@{prefix}-import-error-tip {
+        margin: 4px 14px 0;
+        font-size: 0.75rem;
+        font-weight: 400;
+        color: var(--mui-palette-error-main);
+    }
+    .@{prefix}-file-tip {
+        margin: 0;
+        font-size: 12px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: normal;
+        color: var(--purple-8);
+    }
+
+    .@{prefix}-file-list {
+        padding: 0 10px;
+        list-style-type: none;
+        border: 1px dashed var(--border-color-base);
+
+        li {
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+            padding: 3px;
+
+            span {
+                display: flex;
+                align-items: center;
+
+                .@{prefix}-file-name {
+                    padding: 0;
+                    margin-top: 0;
+                    font-size: 12px;
+                    font-style: normal;
+                    font-weight: 400;
```

Comment:
use less variable instead (`@font-weight-base`). and the default weight is `400`, it may be useless.

Reply:
you're right. it's useless.